src/docs/blueprint_download.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Eclipse Theia Blueprint
+---
+
+# Eclipse Theia Blueprint
+
+<img src="/theia-logo-blueprint.png" alt="Theia Logo Blue" style="max-width: 525px">
+
+## What is Eclipse Theia Blueprint?
+
+Eclipse Theia Blueprint is a **template** for building desktop-based products based on the Eclipse Theia platform, as well as to showcase Eclipse Theia capabilities. TIt is made up of a  subset of existing Eclipse Theia features and extensions and can be easily downloaded and installed on all major operating system platforms (see below). Documentation is available to help you customize and build your own Eclipse Theia-based product.
+
+## What is it not?
+
+Eclipse Theia Blueprint is ***not*** **a production-ready product**. Therefore, it is also not meant to be a replacement for Visual Studio Code or any other IDE.
+
+## Downloads
+
+<span style="color:red">Warning: Currently Theia Blueprint product is an **alpha** version. Please consider the current versions as experimental. </span>
+
+<table cellspacing="25">
+  <thead>
+    <tr>
+      <th>Windows</th>
+      <th>Linux</th>
+      <th>macOS</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="https://www.eclipse.org/downloads/download.php?file=/theia/1.9.1/windows/Theia-1.9.1.exe&r=1">Download 1.9.1</a></td>
+      <td><a href="https://www.eclipse.org/downloads/download.php?file=/theia/1.9.1/linux/Theia-1.9.1.AppImage&r=1">Download 1.9.1</a></td>
+      <td><a href="https://www.eclipse.org/downloads/download.php?file=/theia/1.9.1/macos/Theia-1.9.1.pkg&r=1">Download 1.9.1</a></td>
```

Comment:
Can we not instead point to the `latest` link, this would mean we would not need to maintain the documentation on each update. 

For example `1.9.1` is actually outdated before even being merged:

- https://download.eclipse.org/theia/latest/linux/
- https://download.eclipse.org/theia/latest/macos/
- https://download.eclipse.org/theia/latest/windows/

I think pointing to the actual download location rather than executing the download itself would be more appropriate.

Reply:
I think with this approach we will lose download stats, as we have to use the "select a mirror script" for a file (https://wiki.eclipse.org/Project_Download_Stats#ZIP_files)
But I will check if we can publish the files in a way, so we can have direct links to the latest version

Reply:
Fixed with https://github.com/eclipse-theia/theia-blueprint/issues/51